### ISSUE TITLE
"Add to Apple Wallet" not working

### ISSUE THREAD

**Issue by lmarceau** [mozilla-mobile/firefox-ios#12331]
There was a report that adding a pass to Apple wallet from Firefox isn't functioning properly in v106.

See [comment](https://github.com/mozilla-mobile/firefox-ios/issues/8994#issuecomment-1304822564) on issue https://github.com/mozilla-mobile/firefox-ios/issues/8994

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-5218)

**Comment by walinchus** [comment]
Just FYI, this is still happening today. On mobile Firefox, I can't add tickets to my apple wallet. The button just doesn't work. It doesn't give an error. I thought this wouldn't be a big deal, but twice I tried going into a stadium just showing the ticket on the Firefox browser and the wifi there was so slow that it wouldn't load. (At two different stadiums.) 

I tried downloading the ticketmaster app, which bizarrely does not show my accepted tickets though I am signing in under the same email each time. 

Long story short, I downloaded Chrome for iOS specifically to add tickets to my apple wallet so I can actually get in the stadium this time. But I wanted to flag this as I'm sure I'm not the only one this is happening to.

**Comment by jgh9** [comment]
Most recent updates have fixed issue and was able to add tickets without issue!

**Comment by walinchus** [comment]
Yay thanks!

**Comment by Elliott599** [comment]
I can confirm that this HAS NOT been fixed in v125.3.  I get this error message when trying to add a pass to the Apple Wallet: "Failed to add pass, an error occurred while adding the pass to Wallet.  Please try again later."

If I set Safari as the default browser it works as intended.
thank you.

**Comment by cpeterso** [comment]
I can reproduce in Fx 126.1 on theparkingspot.com.

**Comment by ibookstein** [comment]
Reproduced for me on Firefox 131.0 (45816).

**Comment by data-sync-user** [comment]
➤ gangadi commented:

That is part of #1, Webview refactor

**Comment by data-sync-user** [comment]
➤ Matt P commented:

The corresponding GH issue for the above Jira issue is  https://github.com/mozilla-mobile/firefox-ios/issues/24432 ( https://github.com/mozilla-mobile/firefox-ios/issues/24432|smart-link )
